frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -91,11 +92,11 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
 
   const changeFilter = (values: IPropertyFilter) => {
     const agencyIds = (values.agencies as any)?.value ?? '';
-    setPropertyFilter({ ...values, agencies: agencyIds });
     onChange?.({ ...values, agencies: agencyIds });
   };
 
   const resetFilter = () => {
+    setClear(true);
```

Comment:
Won't it be stuck as `true` after you click the **reset** button?

Reply:
It is setback to false in the Typeahead component!